examples/combined_ODNP.py

```diff
@@ -0,0 +1,333 @@
+from numpy import *
+from pyspecdata import *
+import os
+import sys
+import SpinCore_pp
+import time
+from Instruments import power_control
+from datetime import datetime
+from SpinCore_pp.verifyParams import verifyParams
+from SpinCore_pp.power_helper import gen_powerlist
+from pyspecdata.file_saving.hdf_save_dict_to_group import hdf_save_dict_to_group
+import h5py
+# {{{ from run_Hahn_echo_mw.py
+fl = figlist_var()
+save_file=True
+# {{{ experimental parameters
+# {{{ these need to change for each sample
+output_name = '150uM_TEMPOL'
+IR_postproc = 'spincore_IR_v1'
```

Comment:
this should be called `postproc_type` -- pyspecdata looks for a property called `postproc_type`

Reply:
see [pyspecdata docs](https://jmfrancklab.github.io/pyspecdata/modules.html#pyspecdata.load_files.find_file)

Reply:
also, this does not change every time.

Reply:
edited first comment in this thread

Reply:
Okay so I read the pyspecdata docs and see it isn't a acquisition parameter but it's own property so I keep this structure and later set_prop('postproc_type') = IR_postproc